git_workflow_quick_reference.md

```diff
@@ -0,0 +1,159 @@
+# Developer Workflow Quick-Reference
+
+## Branch Organization
+
+Main branch(es)
+- Usually named `master` and/or `develop`
+  - The `develop` branch is usually the most up-to-date version of the project.
+  - The `master` branch is usually the most recent "stable" release of the project.
+- Contains only reviewed changes
+- Starting point for new features
+- Generally do not ever commit or push directly to these branches.
+
+Feature branches
+- Named after the feature or person (e.g. phil-profile)
+- Used to keep work in progress off of the main branches
+- Should be closed after no longer applicable (feature complete, reviewed, and merged into master/develop)
+- These are where you commit your personal changes
+
+When starting a new project
+- Agree on the “main” branch(es) (`master`, `develop`, etc.)
+- Agree on feature branch naming conventions ([name]/[jira number]/[description])
+- Make sure there are both local and remote copies of the “main” branch and your personal/feature branches
+- Agree on pull request flow (who’s reviewing, who merges when complete)
+
+## Workflow for a Single Feature
+
+### The beginning (no local changes)
+Update your main branch by pulling from the remote
+
+```
+git checkout develop
+git pull origin develop
+```
+
+Create a new feature branch
+```
+git checkout -b phil/JIRA-1234/profile-page
+```
+
+Now you are on your feature branch with the most recent version!
+Begin making your changes!
+
+### Done with personal changes
+
+First Add the files you wish to change.
+- Add specific files like so:
+```
+git add SomeFile.java SomeOtherFile.java
+```
+- Or add all files reachable from the current directory like so:
```

Comment:
I think I agree with this. I'd also say that once the Intrepoid iOS projects are standardized, the `.gitignore` file should include most of those by default (not xibs, but the usual culprits)